INPUT_SCHEMA.json

```diff
@@ -14,7 +14,7 @@
             "title": "Input URL",
             "type": "array",
             "description": "(Required if you don't use 'Search Queries') Here you can provide a list of search URLs.",
-            "editor": "stringList"
+            "editor": "requestListSources"
```

Comment:
This is a breaking change so we need to be careful and probably add a big warning to the CHANGELOG. I think it will break only few users so it is ok but we need to be transparent.

Reply:
@metalwarrior665 Yeah, you are right.
I should just add this change to CHANGELOG as usual, or I need to highlight it in some specific special way?

Reply:
I would add `BREAKING` before it and explain that you now need to provide list of requests
[{ "url": "https://example.com" }] instead of list of strings ["https://example.com"]

Reply:
Actually, one more idea that will help people who use API. If they provide a list of strings, we can convert it to list of requests

Reply:
Good Idea) 
I can make it. 
Shall I create new Issue for it?

Reply:
Actually, let's leave it be until someone complains :) I want to merge this